conditional_operator.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Conditional Operator</title>
+  <script>
+    var amount = 1000;
+    var exchangeRate = 19.50;
+    var total = ( exchangeRate > 0 ) ? amount * exchangeRate : amount;
```

Comment:
What if the exchange rate is 0? or less than 0? write more lines with that scenario.